metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -30,6 +30,8 @@
 %define metron_root         %{_prefix}/%{base_name}
 %define metron_home         %{metron_root}/%{full_version}
 
+%define _binaries_in_noarch_packages_terminate_build   0
```

Comment:
Some of the Node dependencies have shell scripts.  The RPM builder would complain that we are adding shell scripts that need /bin/sh into a 'noarch' RPM.  I had to turn this check off to get the RPMs to build.  The other alternative would be to make this an arch-specific RPM, but that doesn't seem necessary to me.